pkg/build/api/v1/types.go

```diff
@@ -626,6 +691,17 @@ type GenericWebHookEvent struct {
 type GitInfo struct {
 	GitBuildSource    `json:",inline"`
 	GitSourceRevision `json:",inline"`
+
+	// Refs is a list of GitRefs for the provided repo - generally sent
+	// when used from a post-receive hook. This field is optional and is
+	// used when sending multiple refs
+	Refs []GitRefInfo `json:",omitempty"`
```

Comment:
if you're going to change this to an array you should remove the existing fields (and we'll need a conversion) but i'd like to understand why we need this first.